srcpkgs/perl-Text-Template/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'perl-Text-Template'
+pkgname=perl-Text-Template
+version=1.60
+revision=1
+wrksrc="${pkgname/perl-/}-${version}"
+build_style=perl-module
+hostmakedepends="perl"
+makedepends="perl"
+depends="perl"
```

Comment:
Add `perl-Test-Warnings` to `checkdepends`:

> `t/warnings.t ........... skipped: [ Test::Warnings ] is required for testing`

Reply:
Fixed, but getting this:
```
t/template-encoding.t .. skipped: [ Test::More::UTF8 ] is required for testing
t/warnings.t ........... ok
```

It seems Void has no perl-Test-More package.

Reply:
Hmm, but this worls:

`perl -MTest::More::UTF8`.

Weird.

Reply:
Very weird, adding perl-Test-Simple to checkdepends, makes the build fail. Any idea why ?

I also cannot install perl-Test-Simple:

```
➜  sudo xbps-install -Syu perl-Test-Simple
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/x86_64-repodata' ...
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/multilib/nonfree/x86_64-repodata' ...
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/multilib/x86_64-repodata' ...
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/nonfree/x86_64-repodata' ...
[*] Updating repository `https://alpha.de.repo.voidlinux.org/current/debug/x86_64-repodata' ...
perf-5.10.4_6: broken, unresolvable shlib `libperl.so.5.34'
perl-Cairo-1.107_3: broken, unresolvable shlib `libperl.so.5.34'
perl-Cairo-GObject-1.005_4: broken, unresolvable shlib `libperl.so.5.34'
perl-Class-XSAccessor-1.19_8: broken, unresolvable shlib `libperl.so.5.34'
perl-Curses-1.36_4: broken, unresolvable shlib `libperl.so.5.34'
perl-DBD-SQLite-1.66_3: broken, unresolvable shlib `libperl.so.5.34'
perl-DBI-1.643_3: broken, unresolvable shlib `libperl.so.5.34'
perl-Data-MessagePack-1.00_7: broken, unresolvable shlib `libperl.so.5.34'
perl-Devel-NYTProf-6.10_1: broken, unresolvable shlib `libperl.so.5.34'
perl-Glib-1.3293_2: broken, unresolvable shlib `libperl.so.5.34'
perl-Glib-Object-Introspection-0.049_2: broken, unresolvable shlib `libperl.so.5.34'
perl-HTML-Parser-3.76_1: broken, unresolvable shlib `libperl.so.5.34'
perl-JSON-XS-4.03_3: broken, unresolvable shlib `libperl.so.5.34'
perl-Net-SSLeay-1.90_3: broken, unresolvable shlib `libperl.so.5.34'
perl-Package-Stash-XS-0.29_4: broken, unresolvable shlib `libperl.so.5.34'
perl-Params-Util-1.102_1: broken, unresolvable shlib `libperl.so.5.34'
perl-Sub-Identify-0.14_7: broken, unresolvable shlib `libperl.so.5.34'
perl-Sub-Name-0.26_3: broken, unresolvable shlib `libperl.so.5.34'
perl-Term-ReadLine-Gnu-1.42_2: broken, unresolvable shlib `libperl.so.5.34'
perl-Unicode-LineBreak-2019.001_4: broken, unresolvable shlib `libperl.so.5.34'
perl-Variable-Magic-0.62_7: broken, unresolvable shlib `libperl.so.5.34'
perl-XML-LibXML-2.0134_7: broken, unresolvable shlib `libperl.so.5.34'
vim-huge-8.2.3565_1: broken, unresolvable shlib `libperl.so.5.34'
Transaction aborted due to unresolved shlibs.
```

Reply:
You shouldn’t have to add `perl-Test-Simple` because it’s provided by perl itself:
```
$ corelist Test::Simple

Data for 2021-05-20
Test::Simple was first released with perl v5.6.2
```

The above error is because `perl` provides `perl-Test-Simple` (see the perl template) and installing `perl-Test-Simple` tries to replace the `perl` package with it, but the packages listed above depend on `libperl.so.5.34` which would then be missing and therefore `xbps-install` aborts.

The stand-alone `perl-Test-Simple` package has been removed in https://github.com/void-linux/void-packages/commit/ef5218bc12484eab1ecfcd6527eb07da39a0896b but was later re-added in https://github.com/void-linux/void-packages/commit/78d94807596081fa332c799058ca1a94a871d6d6; don’t know why, maybe to have an up-to-date version for some reasons; perhaps @zdykstra can shed some light on this. ;)

As for `perl -MTest::More::UTF8`: you probably have it installed somewhere on your system, but there is no Void package. So if you want to use it here, you have to package it too, and add it to this PR as a separate commit.

Reply:
I don't think there's any reason for perl-Test-Simple to be provided as a separate package any more. The version packaged now is identical to the version provided by perl_core. I picked the package up after it was orphaned; I didn't dig any deeper into why it was actually packaged.